pypac/api.py

```diff
@@ -145,8 +152,9 @@ class PACSession(requests.Session):
     """
 
     def __init__(self, pac=None, proxy_auth=None, pac_enabled=True,
-                 response_proxy_fail_filter=None, exception_proxy_fail_filter=None,
-                 socks_scheme='socks5', **kwargs):
+                 response_proxy_fail_filter=None,
+                 exception_proxy_fail_filter=None, socks_scheme='socks5',
+                 session=None, **kwargs):

```

Comment:
This new `session` isn't used.

Reply:
Yes, it was my mistake. I did not add class casting to the `__init__` that uses session:
`session.__class__ = PACSession`
Maybe `if session:` is not good enough and is better to add `if isinstance(session, requests.Session):` and raise exception if `False`